docs/en/siem/rules-ui-create.asciidoc

```diff
@@ -0,0 +1,141 @@
++
+The *Create new rule* page is displayed.
+[role="screenshot"]
+image::create-new-rule.png[]
+. Define which {es} indices the rule analyzes for signals.
+. Use the filter and query fields to create the criteria used for detecting 
+signals. For example, a rule that detects when a Microsoft Office product 
+starts a process in a user folder uses the following query and filters:
+
+* _Custom query_ `process.parent.name: EXCEL.EXE or process.parent.name: MSPUB.EXE or process.parent.name: OUTLOOK.EXE or process.parent.name: POWERPNT.EXE or process.parent.name: VISIO.EXE or process.parent.name: WINWORD.EXE`
+* _Add filter_:
+** *Field*: `event.action`
+** *Operator*: `is`
+** *Value*: `Process Create (rule: ProcessCreate)`
+. Click *Continue*.
++
+The *About rule* pane is displayed.
+[role="screenshot"]
+image::about-rule-pane.png[]
+. Fill in the following fields:
+.. _Name_: The rule's name.
+.. _Description_: A description of what the rule does.
+.. _Severity_: Select the severity levels of signals created by the rule:
+* `Low`: Signals that are of interest but generally not considered to be 
+security incidents. Sometimes, a combination of low severity events can 
+indicate suspicious activity.
+* `Medium`: Signals that require investigation.
+* `High`: Signals that require an immediate investigation.
+* `Critical`: Signals that indicate a security incident has occurred.
+.. _Risk score_: A numerical value between 0 and 100 that correlates with the _Severity_ level. General guidelines are:
+* `21` represents low severity.
+* `47` represents medium severity.
+* `73` represents high severity.
+* `99` represents critical severity.
+.. For additional options, click *Advanced settings* and fill in any of 
+these fields: 
+... _Timeline template_ (optional): Select a timeline template used when you 
+send a signal created by the rule to the timeline.
++
+TIP: Before you create rules, create and save relevant
+<<timelines-ui,timelines>> so they can be selected here. 
+
+... _Reference URLs_ (optional): References to information that is relevant to 
+the rule. For example, links to relevant background information.
+... _False positives_ (optional): List of common scenarios that may produce 
+false-positive signals.
+... _MITRE ATT&CK^TM^_ (optional): Relevant MITRE framework tactics and techniques.
+... _Tags_ (optional): Words and phrases used to categorize, filter, and search 
+the rule.
+. Click *Continue*.
++
+The *Schedule rule* pane is displayed.
+[role="screenshot"]
+image::schedule-rule.png[]
+. Select how often the rule runs.
+. Optionally, add `look-back` time to the rule. When defined, the rule searches 
+indices with the additional time.
++
+For example, if you set a rule to run every 5 minutes with an additional
+look-back time of 1 minute, the rule runs every 5 minutes but analyses the 
+documents added to indices during the last 6 minutes. This overlap helps ensure 
+signals are issued even when incoming data is delayed.
```

Comment:
Very nice wording of the look-back feature here @benskelker 👏 🙂 This has been a tough one to describe/grok in all the recent demos it seems.

One thing that might be worth noting though is that the server performs de-duplication of signals when it fetches data that has already resulted in the creation of signals.

Also, I believe the reason for this feature is not so much for when incoming data is delayed, but rather when there is a delay in the execution of the rule's query. E.g. if the user turns on 100's of rules and only has a single Kibana instance, rules may sit in the queue to be run long enough that there is a gap in coverage since the last time it was ran, and thus this additional look-back ensures that gap of data is queried (and then de-duped if it queries into the previous run's interval).

Feel free to reach out if you have any questions around the corner cases. I'll do my best to answer, but @FrankHassanabad / @dhurley14 will know for sure though as they implemented this part of the Detection Engine 😅